docs/usage.md

```diff
@@ -21,43 +25,88 @@ You will need to create a samplesheet with information about the samples you wou
+CONTROL,1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz,,
+CONTROL,2,AEG588A2_S2_L002_R1_001.fastq.gz,AEG588A2_S2_L002_R2_001.fastq.gz,,
+CONTROL,3,AEG588A3_S3_L002_R1_001.fastq.gz,AEG588A3_S3_L002_R2_001.fastq.gz,,
+TREATMENT,1,AEG588A4_S4_L003_R1_001.fastq.gz,AEG588A4_S4_L003_R2_001.fastq.gz,,
+TREATMENT,2,AEG588A5_S5_L003_R1_001.fastq.gz,AEG588A5_S5_L003_R2_001.fastq.gz,,
+TREATMENT,3,AEG588A6_S6_L003_R1_001.fastq.gz,AEG588A6_S6_L003_R2_001.fastq.gz,,
+TREATMENT,3,AEG588A6_S6_L004_R1_001.fastq.gz,AEG588A6_S6_L004_R2_001.fastq.gz,,
 ```
 
 | Column         | Description                                                                                                                                                                            |
 |----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `sample`       | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `group`       | Custom group name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `replicate`   | Biological replicates of the samples.  |
 | `fastq_1`      | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
 | `fastq_2`      | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| `md5_1`        | Checksum for fastq_1. The checksums of the files will be check to make sure the file is not truncated if provided.  |
+| `md5_2`        | Checksum for fastq_2. The checksums of the files will be check to make sure the file is not truncated if provided.  |
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+### Config file
+
+Users can also run the pipeline by provide a config file by `-c` parameter (see `-c` section below).
+
+```bash
+nextflow run nf-core/hicar -c 'path/to/the/profile/file' -profile docker
+```
+
+In the config file, the design table, and more options can be defined.
+
+Here is an example profile.config file for running nf-core/hicar on slurm cluster.
+
+```groovy
+// submit by slurm
+process.executor = "slurm"
+process.clusterOptions = "-J nextFlowHiCAR"
+
+params {
+    config_profile_name = 'profile name here'
+    config_profile_description = 'profile description here'
+    // report email
+    email = 'users@email.addr'
+    // resources
+    max_cpus   = 8
+    max_memory = 60.GB
+    max_time   = 48.h
```

Comment:
https://github.com/nf-core/hicar/pull/1#discussion_r741471586

Please avoid adding pipeline-specific documentation about general topics such as running on specific hardware (a cluster, slurm etc). It's better to point people to the central documentation on these things, so that we have a single location to keep up to date.

Reply:
I agree. This section will be removed.